src/borg/archive.py

```diff
@@ -1010,7 +1010,7 @@ def init_chunks(self):
         """
         # Explicitly set the initial hash table capacity to avoid performance issues
         # due to hash table "resonance"
-        capacity = int(len(self.repository) * (1/0.93) + 1)  # > len * 1.0 / HASH_MAX_LOAD (see _hashindex.c)
+        capacity = int(len(self.repository) * (1/0.99) + 1)  # > len * 1.0 / HASH_MAX_LOAD (see _hashindex.c)
```

Comment:
referring to my comment in previous changeset: iirc we made the load factor a global name in hashindex.pyx in master branch, you just need to rebase upon that at some time, no need to re-implement this.